front/src/applications/operationalStudies/views/ManageTimetableItem.tsx

```diff
@@ -228,10 +229,7 @@ const ManageTimetableItem = () => {
     []
   );
 
-  const { data: { results: subCategories } = { results: [] } } =
-    osrdEditoastApi.endpoints.getSubCategory.useQuery({
-      pageSize: 100,
-    });
+  const { subCategories } = useSubCategoryContext();
```

Comment:
I think that now you have to get the subcategories without the `{}`
```suggestion
  const subCategories = useSubCategoryContext();
```

Reply:
Done